src/useResizable.ts

```diff
@@ -45,16 +46,26 @@ const useResizable = ({
 
       const currentPosition = (() => {
         if (axis === 'x') {
+          if (containerRef && containerRef.current) {
+            const containerNode = containerRef.current;
+            const { left } = containerNode.getBoundingClientRect();
+            return reverse ? left - e.clientX : e.clientX - left;
+          }
           return reverse ? document.body.offsetWidth - e.clientX : e.clientX;
         }
+        if (containerRef && containerRef.current) {
+          const containerNode = containerRef.current;
+          const { top } = containerNode.getBoundingClientRect();
+          return reverse ? top - e.clientY : e.clientY - top;
+        }
         return reverse ? document.body.offsetHeight - e.clientY : e.clientY;
```

Comment:
@KlonD90 
This part doesn't seem to work well when `reverse === true`.
I think the size of the container element needs to be calculated.
```ts
return reverse ? height + top - e.clientY : e.clientY - top;
```

Reply:
Thank you so much for feedback!